hazelcast/src/main/java/com/hazelcast/map/impl/record/CachedDataRecord.java

```diff
@@ -18,10 +18,15 @@
 
 import com.hazelcast.nio.serialization.Data;
 
+import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
+
 /**
  * CachedDataRecord.
  */
 class CachedDataRecord extends DataRecord {
+    private static final AtomicReferenceFieldUpdater<CachedDataRecord, Object> CACHED_VALUE_UPDATER =
```

Comment:
Just a flavor thing; I always removed the _UPDATER section. It is already clear from the CACHED_VALUE you are talking to a constant. And I find this a bit easier to read:

CACHED_VALUE.compareAndSet(this, ....)

But this is flavor.